src/pattern-matching/destructuring-arrays.md

```diff
@@ -5,3 +5,10 @@ You can destructure arrays, tuples, and slices by matching on their elements:
 ```rust,editable
 {{#include ../../third_party/rust-by-example/destructuring-arrays.rs}}
 ```
+
+<details>
+  
+Key points:
+* Continuing our journey in pattern matching, this slide just shows another type that can destructured with pattern matching. 
+* This is a another great time to do some live coding. You can add/remove values from the array, compare the syntax `..` and `_`, and share any errors you get.
```

Comment:
Can you update this with precise instructions on what to do?

Please make sure that the notes _add_ to the slide. They should explain things to people who don't have an instructor around as well as tell the instructor what they should focus on.